tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -525,11 +525,22 @@ Get List Of All Available Container Size
     [Return]    ${size}
 
 Get Previously Selected Notebook Image Details
+    [Documentation]  Returns image:tag information from the Notebook CR for a user
+    ...    or minimal-gpu:default if the CR doesn't exist
```

Comment:
This kw is currently customized for one test case. Could you add a parameter with the username? Or move the kw to the test suite?

Reply:
moved to test suite